rules-reviewed/eap6/websphere/websphere-catchall.windup.xml

```diff
@@ -83,7 +83,7 @@
                         </not>
                     </when>
                     <perform>
-                        <hint title="IBM ILog proprietary type reference" effort="0" category-id="potential">
+                        <hint title="IBM ILog proprietary type reference" effort="1" category-id="potential">
```

Comment:
Let's leave "0" for the moment as discussed in our conf-call several weeks ago.

In the future, it needs to be clear that all issues flagged as "potential". As such they potentially encompass the highest efforts and risks.